docs/querying/basics.md

```diff
@@ -4,17 +4,15 @@ nav_title: Basics
 sort_rank: 1
 ---
 
-# Querying Prometheus
+# Querying basics
 
 Prometheus provides a functional query language called PromQL (Prometheus Query
-Language) that lets the user select and aggregate time series data in real
+Language) that lets you select and aggregate time series data in real
```

Comment:
The problem with passive voice is that it is less clear than active voice and uses more words. Best practice in technical writing is to use active voice whenever possible.

Reply:
Our docs are primarily passive voice rather than 2nd person, and if that's to be changed that would need to be discussed more broadly.

Reply:
My $0.02: active voice here feels more engaging as if the docs are talking to "me", so maybe we could gradually shift all docs to active voice?

Reply:
One way or the other making piecemeal changes to our doc style is not the way to do that. As is stands this is not our style, so PRs should be looking to eliminate rather than propagate it.

Reply:
Where is this stated? I have not found a style guide for documentation with any sort of grammar or formating rules for this project.

Reply:
I looked at what was common in this doc and the related ones, consistency is enough of a reason.

Reply:
I agree with @brian-brazil

Reply:
Reverted

Reply:
But...consistent? Really? Here are some examples of the second-person already existing in your docs:

https://prometheus.io/docs/practices/histograms/#count-and-sum-of-observations
  ` In principle, however, you can use summaries and histograms to observe negative values (e.g. temperatures in centigrade). In that case, the sum of observations can go down, so you cannot apply rate() to it anymore.`

https://prometheus.io/docs/practices/histograms/#quantiles
`You can use both summaries and histograms to calculate so-called φ-quantiles, where 0 ≤ φ ≤ 1. `

https://prometheus.io/docs/prometheus/latest/configuration/configuration/
`You may wish to check out the 3rd party Prometheus Operator, which automates the Prometheus setup on top of Kubernetes.`

I could go on, but I think you get the idea.

I'm not even going to touch all the many examples of active voice that are already in the Prometheus docs. Point is, you are not consistent. Why not allow people to improve the docs and eventually move the needle towards "better"?